test/hotspot/jtreg/TEST.groups

```diff
@@ -415,7 +415,8 @@ tier1_runtime = \
  -runtime/Unsafe/RangeCheck.java \
   sanity/ \
  -:tier1_runtime_appcds_exclude \
- -runtime/signal
+ -runtime/signal \
+ -runtime/stack
```

Comment:
So what tier will run these now? Are these really stress tests?

Reply:
Good question.  They are stress tests and not something we want to run in tier1.

Reply:
These tests are executed in tier2 now. They are stress tests, trying to provoke SOE in different ways. They don't need a lot of resources but might take a long time which is not recommended for tier1 tests.